src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    color: $text-color-main-accent;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $text-color-secondary;
+    font-style: normal;
+    font-weight: 400;
```

Comment:
font-style: normal and  font-weight: 400 is default values, so it's declaration is redundant